ARCs/arc-0007.md

```diff
@@ -86,11 +86,11 @@ Regarding a call to `postTxns(stxns)` with promised return value `ret`:
   > Another such use case is when the user uses a specific trusted node that has some legal restrictions.
 * `postTxns` **MUST** wait for confirmation that the transactions are finalized.
   > TODO: Decide whether to add an optional flag to not wait for that.
-* If successful, `postTxns` **MUST** resolve the returned promise with the list of transaction IDs `txnIds` of the posted transactions `stxn`.
+* If successful, `postTxns` **MUST** resolve the returned promise with the list of transaction IDs `txnIDs` of the posted transactions `stxn`.
 * If unsuccessful, `postTxns` **MUST** reject the promise with an error `err` of type `PostTxnsError` such that:
-  * `err.code=4400` if there was a failure sending the transactions or a code as specified in [ARC-0001](arc-0001.md) if the user or function disallowed posting the transactions.
+  * `err.code=4400` if there was a failure sending the transactions or a code as specified in [ARC-0001](arc-0001.md#error-standards) if the user or function disallowed posting the transactions.
   * `err.message` **SHOULD** describe what went wrong in as much detail as possible.
-  * `err.successTxnIds` **MUST** be an array such that `err.successTxnId[i]` is the transaction ID of `stxns[i]` if `stxns[i]` was succesfully committed to the blockchain, and `null` otherwise.
+  * `err.successTxnIDs` **MUST** be an array such that `err.successTxnID[i]` is the transaction ID of `stxns[i]` if `stxns[i]` was succesfully committed to the blockchain, and `null` otherwise.
```

Comment:
```suggestion
  * `err.successTxnIDs` **MUST** be an array such that `err.successTxnID[i]` is the transaction ID of `stxns[i]` if `stxns[i]` was successfully committed to the blockchain, and `null` otherwise.
```